### ISSUE TITLE
setColumnRemarks for mariaDB is not supported

### ISSUE THREAD

**Issue by janjicandjelka8** [liquibase/liquibase#1794]
## Environment
**Liquibase Version**: 3.6.2

**Liquibase Integration & Version**: springboot 2.0.2 and gradle

**Liquibase Extension(s) & Version**: /

**Database Vendor & Version**: MariaDB 10.5.9

**Operating System Type & Version**: Windows 10 Enterprise

## Description
If I add setColumnRemarks changeset, it throws an error for MariaDB. The example of my changeset is like following:

```
<changeSet id="abc_pk_id_remarks_mariadb" author="userABC" dbms="mariadb">
    <setColumnRemarks
      columnName="id"
      remarks="Internal identifier of ABC"
      tableName="abc"/>
  </changeSet>
```

## Steps To Reproduce
* add setColumnRemarks changset and try to execute it against MariaDB

## Actual Behavior
* It throws validation error:

```
1 change have validation failures 
setColumnRemarks is not supported on mariadb, 0_1_0/abc.xml::abc_pk_id_remarks_mariadb::userABC
```

## Expected/Desired Behavior
There is no errors if setColumnRemarks tag is used for changing column remarks on MariaDB. 
In official liquibase documentation, it is stated the setColumnRemarks is supported by MariaDB but it doesn't seem so.

┆Issue is synchronized with this [Jira Bug](https://datical.atlassian.net/browse/LB-1430) by [Unito](https://www.unito.io)

**Comment by molivasdat** [comment]
Hi @janjicandjelka8 Thanks for creating this issue.  We will add it to the list of issues that we are processing.  We will be adding it to a 4.x release or later and not into the 3.6.x branch.

**Comment by nvoxland** [comment]
This was fixed in 4.3. Double checked with your sample changelog.

It does require setting the `columnDataType` attribute because mariadb requires a general ALTER COLUMN to add the column and that requires the datatype.
